examples/adaptive_library/DXFImport.py

```diff
@@ -0,0 +1,205 @@
+"""
+Custom DXF Geometry
+===================
+This script applies the adaptive templates functionality to import custom rotor geometry from a
+.dxf file.
+"""
+# %%
+# .. note::
+#    For more information on how to add imported DXF geometries to Adaptive Templates in Motor-CAD,
+#    see :ref:`ref_adaptive_templates_UG` in the :ref:`ref_user_guide`.
+#
+# .. note::
+#    DXF Import for Adaptive Templates in Motor-CAD requires v2024.2.0.6 (Motor-CAD 2024 R2) or
+#    later and PyMotorCAD v0.4.2. To update PyMotorCAD in Motor-CAD, go to
+#    **Scripting -> Settings** and select 'Update to Latest Release' under PyMotorCAD updates.
+
+# %%
+# This script is designed to be run from Motor-CAD template "e8". The "e8" template file contains an
+# imported custom geometry from a .dxf file. The template should be adjusted slightly to more
+# closely match the imported custom geometry before running the Adaptive Templates script.
+# The following Standard Template geometry changes should be made:
+#
+# * Set L1 Mag Gap Inner and L2 Mag Gap Inner to 0 mm (for Magnet Layers 1 and 2).
+#
+# * Set L2 Web Thickness to 18 mm (for Magnet Layer 2).
+#
+# * Set L2 Pole Arc [ED] to 105 electrical degrees (for Magnet Layer 2).
+#
+# * Set L2 RDuct Rad Dia to 123.8 mm (for Rotor Duct Layer 2).
+#
+# * Set L2 RDuct Dia to 4.94 mm (for Rotor Duct Layer 2).
+
+# %%
+# .. image:: ../../images/DXFImport1_2.png
+
+# %%
+# If no Motor-CAD file is open, the e8 template will be loaded and the geometry will be adjusted as
+# described above.
```

Comment:
```suggestion
# If no Motor-CAD file is open, the e8 template is loaded and the geometry is adjusted as
# described earlier.
```
Earlier, you placed the e8 in quotation marks.  We need to be consistent--and following formatting guidelines.